Bluetooth: A2DP: add delay report at A2DP sink.

gzh-terry opened the pull request:
- This patch allows to send delay report as A2DP sink, which is mandatory since A2DP v1.2.

- Add delay report service when receiving GetAllCapabilities command at a SNK SEP.

gzh-terry committed:
Bluetooth: A2DP: add delay report at A2DP sink.

- This patch allows to send delay report as A2DP sink, which is mandatory since A2DP v1.2.

- Add delay report service when receiving GetAllCapabilities command at a SNK SEP.

Signed-off-by: Zihao Gao <gaozihao@xiaomi.com>

MarkWangChinese commented:
Hi @gzh-terry I implemented the delay report in https://github.com/zephyrproject-rtos/zephyr/pull/95045 I implemented the source side too to test the feature with two boards. Could you help to review it? Let's algin to use one PR to enable the delay report feature.

gzh-terry commented:
> Hi @gzh-terry I implemented the delay report in #95045 I implemented the source side too to test the feature with two boards. Could you help to review it? Let's algin to use one PR to enable the delay report feature.

Good to see your progress.

This PR can be closed.
